packages/angular/README.md

```diff
@@ -53,19 +51,14 @@ This library is providing an `NgModule` named `SynergyModule`, which takes care
 
 ```typescript
 // src/app/app.module.ts
-import { NgModule } from '@angular/core';
-import { BrowserModule } from '@angular/platform-browser';
-import { AppComponent } from './app.component';
-import { SynergyComponentsModule } from '@synergy-design-system/angular';
+import { NgModule } from "@angular/core";
+import { BrowserModule } from "@angular/platform-browser";
+import { AppComponent } from "./app.component";
+import { SynergyComponentsModule } from "@synergy-design-system/angular";
 
 @NgModule({
-  declarations: [
```

Comment:
I would leave the formatting here alone, as it makes it easier to copy lines and see the structure of the config.